README.md

```diff
@@ -175,12 +69,14 @@ You can use the built-in `go test` command to run the tests.
 To run all tests in the current directory, you can run:
 
 ```shell
-go test -v ./...
+go test -v ./... -timeout=<timeout>m
 ```
 
+**Note:** The timeout is set to 10 minutes by default. Make sure to set a timeout that is long enough to complete the test.
```

Comment:
This is different than the timeout in the knuu options right? if so, let's call that out explicitly to avoid confusion.

Reply:
good point. I would also like to remove the `go test` stuff as they are not really part of knuu. So we can remove the entire thing to avoid confusion and add more explanation on knuu timeout.